### PR TITLE
Bump to v0.4.0

### PR DESCRIPTION
**Pull request by rai-nhdaly**

### CODE REVIEW

**Comment by rai-nhdaly** [comment]
CC: @li1

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "Blobs"
 uuid = "2063b8be-a4ad-11e8-1fb9-93d01944c8ed"
 authors = ["Jamie Brandon <jamie@scattered-thoughts.net>"]
```
Comment (li1) [inline Project.toml:3]: Haha, just to not review without leaving a comment — do we want to keep this in?

Comment (rai-nhdaly) [reply Project.toml:3]: 🤷  It's maybe nice to recognize the work he did early on it?
But also i think it's fine to delete it yeah.

**Comment by rai-nhdaly** [comment]
gonna merge for now. we can always address your comment in 1.0 release :p

**Pull request merged**